### ISSUE TITLE
Autokick Idlers

### ISSUE THREAD

**Issue by jibarofoss** [WE-Kaito/digimon-tcg-simulator#167]
Way too many idle users on Lobby, clogging up the people that actually want to play. A 5-10 min autokick timer should suffice.

**Comment by Realsoter** [comment]
Hardly disagree. I want stay in lobby for as long as I want until I get a request when I feel like playing.

**Comment by jibarofoss** [comment]
> Hardly disagree. I want stay in lobby for as long as I want until I get a request when I feel like playing.

Nah. At all times, there are at least 2 idlers. You miss actual players trying to invite idlers to a game.
